web/oauth_test.go

```diff
@@ -65,7 +65,7 @@ func TestAuthorizeOAuthApp(t *testing.T) {
 	}
 
 	rapp, appErr := th.App.CreateOAuthApp(oapp)
-	CheckNoAppError(t, appErr)
+	require.Nil(t, appErr, "Expected no error, got %q", appErr.Error())
```

Comment:
This will panic. You need to just use 
```suggestion
	require.Nil(t, appErr)
```